webdriver/tests/element_clear/clear.py

```diff
@@ -121,7 +120,7 @@ def test_keyboard_interactable(session):
                           ("time", "19:48", ""),
                           ("month", "2017-11", ""),
                           ("week", "2017-W52", "")])
-def test_input(session, add_event_listeners, tracked_events, type, value, default):
+def test_input(session, add_event_listeners, tracked_events, inline, type, value, default):
```

Comment:
Mind moving the global fixture before all other locally defined ones? So right after `session`. Applies to other cases as well.